pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -133,7 +138,8 @@ public QueryExecutor openConnectionImpl(HostSpec[] hostSpecs, String user, Strin
     Iterator<HostSpec> hostIter = hostChooser.iterator();
     while (hostIter.hasNext()) {
       HostSpec hostSpec = hostIter.next();
-      LOGGER.log(Level.FINE, "Trying to establish a protocol version 3 connection to {0}", hostSpec);
+      LOGGER.log(Level.FINE, "Trying to establish a protocol version 3 connection to {0}",
```

Comment:
I did so because of the (strict) checkstyle requirements, which I only noticed after Travis run its job. I could resubmit the commits, squashing them, but I believe that is not what the history of the feature was, so I think it is more appropriate to leave it as is. Please let me know otherwise